### PR TITLE
Update mongodb-atlas.md

### PR DESCRIPTION
**Pull request by ashni-mehta**
Adding prerequisites and more context on MongoDB Atlas.

## Issue

Closes #

## Change
Updating MongoDB Atlas docs.

## General checklist

- [ ] There are no breaking changes
- [ ] I have added unit and integration tests for my change
- [ ] I have manually run all the unit and integration tests in the module I have added/changed, and they are all green
- [ ] I have manually run all the unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules, and they are all green

- [x] I have added/updated the [documentation](https://github.com/langchain4j/langchain4j/tree/main/docs/docs)
- [ ] I have added an example in the [examples repo](https://github.com/langchain4j/langchain4j-examples) (only for "big" features)
- [ ] I have added/updated [Spring Boot starter(s)](https://github.com/langchain4j/langchain4j-spring) (if applicable)

## Checklist for adding new model integration

- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml)

## Checklist for adding new embedding store integration

- [ ] I have added a `{NameOfIntegration}EmbeddingStoreIT` that extends from either `EmbeddingStoreIT` or `EmbeddingStoreWithFilteringIT`
- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml)

## Checklist for changing existing embedding store integration

- [ ] I have manually verified that the `{NameOfIntegration}EmbeddingStore` works correctly with the data persisted using the latest released version of LangChain4j

### CODE REVIEW

File: docs/docs/integrations/embedding-stores/mongodb-atlas.md [hunk 1]
```diff
@@ -2,10 +2,17 @@
 sidebar_position: 10
 ---
 
-# MongoDB Atlas
+# MongoDB Atlas and Vector Search
 
-https://www.mongodb.com/atlas
+[MongoDB Atlas](https://www.mongodb.com/docs/atlas/) is a fully-managed cloud database available in AWS, Azure, and GCP. It supports native Vector Search and full text search (BM25) on your MongoDB document data.
 
+[MongoDB Atlas Vector Search](https://www.mongodb.com/products/platform/atlas-vector-search) allows to store your embeddings in MongoDB documents, create a vector search index, and perform KNN search with an approximate nearest neighbor algorithm (Hierarchical Navigable Small Worlds). It uses the $vectorSearch MQL Stage.
```
Comment (rustagir) [inline docs/docs/integrations/embedding-stores/mongodb-atlas.md]:
```suggestion
[MongoDB Atlas Vector Search](https://www.mongodb.com/products/platform/atlas-vector-search) allows you to store your embeddings in MongoDB documents, create vector search indexes, and perform KNN search with an approximate nearest neighbor algorithm called Hierarchical Navigable Small Worlds. You can implement this feature by using the `$vectorSearch` MQL aggregation stage.
```

File: docs/docs/integrations/embedding-stores/mongodb-atlas.md [hunk 2]
```diff
@@ -2,10 +2,17 @@
...
+
+## Prerequisites
+
+You must have an Atlas cluster with either MongoDB version 6.0.11, 7.0.2, or later (including RCs).
```
Comment (rustagir) [inline docs/docs/integrations/embedding-stores/mongodb-atlas.md]:
```suggestion
You must have an Atlas cluster that runs one of the following MongoDB versions:

- 6.0.11
- 7.0.2
- Later versions (including Release Candidates).
```

File: docs/docs/integrations/embedding-stores/mongodb-atlas.md [hunk 3]
```diff
@@ -2,10 +2,17 @@
...
+
+You will also need an Atlas cluster deployed. MongoDB offers a free forever cluster that you can use to try out [MongoDB Atlas](https://www.mongodb.com/docs/atlas/getting-started/). Once you have a cluster, you can create a vector search index via the JSON Editor.
```
Comment (rustagir) [inline docs/docs/integrations/embedding-stores/mongodb-atlas.md]:
```suggestion
To use Atlas Vector search, you need to have an Atlas deployment. MongoDB offers a free forever cluster that you can use for testing. See the [Get Started with Atlas](https://www.mongodb.com/docs/atlas/getting-started/) tutorial to learn more. Once you deploy a cluster, you can create a vector search index by using the index JSON editor.
```

**Review by rustagir — APPROVED** [review]
left some clarifying suggestions!

**Review by ghost — APPROVED** [review]
@ashni-mongodb thank you!

**Pull request merged**